OWNERS.md

```diff
@@ -10,7 +10,7 @@ routing PRs, questions, etc. to the right place.
   * Catch-all, "all the things", and generally trying to make himself obsolete as fast as
     possible.
 * Harvey Tuch ([htuch](https://github.com/htuch)) (htuch@google.com)
-  * APIs, xDS, UDPA, gRPC, configuration, security, Python, and Bash.
+  * xDS APIs, configuration and control plane.
```

Comment:
@phlax happy to help with anything you need in the general docs space as well, but that doesn't need to be called out here.